pyproject.toml

```diff
@@ -1,3 +1,56 @@
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "cubed"
+version = "0.6.0"
+authors = [
+    {name = "Tom White", email = "tom.e.white@gmail.com"},
+]
+license = {text = "Apache License 2.0"}
+description = "Bounded-memory serverless distributed N-dimensional array processing"
+readme = {file = "README.md", content-type = "text/markdown"}
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "aiostream",
+    "fsspec",
+    "mypy_extensions", # for rechunker
```

Comment:
Is this still required now that cubed doesn't depend on rechunker?

Reply:
Yes, the vendored copy of rechunker uses it for `NamedArg`:

https://github.com/tomwhite/cubed/blob/74145fb826d21496ca86b4e64b0fcb1c55d8a931/cubed/vendor/rechunker/types.py#L15

Maybe we could remove that at some point, but it's needed as things currently stand.